src/main/java/com/cursor/demo/entity/Profile.java

```diff
@@ -0,0 +1,44 @@
+package com.cursor.demo.entity;
+
+import lombok.*;
+
+import javax.persistence.*;
+
+@SuppressWarnings("SyntaxError")
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity(name = "Profile")
+@Table(name = "profiles", schema = "public")
+public class Profile {
+    @Id
+    private int id;
+
+    @Column(name = "username")
+    private String userName;
+
+    @Column(name = "job_title")
+    private String jobTitle;
+
```

Comment:
it's bad to have different codestyle. above u use username column for userName variable. few lines below for same styled variable (jobTitle) you use job_title
So please use same style in whole your program please

Reply:
> it's bad to have different codestyle. above u use username column for userName variable. few lines below for same styled variable (jobTitle) you use job_title
> So please use same style in whole your program please

It was fixed